.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.14.4
+      - image: circleci/postgres:9.4.26-ram
+        environment:
+          POSTGRES_DB: gorptest
+          POSTGRES_USER: gorptest
+          POSTGRES_PASSWORD: gorptest
+      - image: circleci/mysql:5.7.30-ram
+        environment:
+          MYSQL_DATABASE: gorptest
+          MYSQL_USER: gorptest
+          MYSQL_PASSWORD: gorptest
+    steps:
+      - checkout
+      - run:
+          name: Run Tests
+          command: |
+            go get -u github.com/onsi/ginkgo/ginkgo
+            ./test_all.sh
```

Comment:
In this bash script there are also remnants of travis and our blacklisting does not recognize the name `test_all.sh`. Personally I would favour converting it into a Makefile to go with the standard, but we can also do this at a later iteration.

Reply:
Ok, I’ll tackle it

Reply:
Done